src/pages/registration/studentID/index.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled'
+import { TextInput } from '@components/basic/InputBox'
+import { FillBtn } from '@components/basic/button/Button'
+import { ChangeEvent, useState, useEffect } from 'react'
+import { withRouter, NextRouter, useRouter } from 'next/router'
+import { request } from '@utils/api'
+import { PutStudentIDParams, PutStudentIDResults } from '@api/insertStudentID'
+import { LoadUserInfoParams, LoadUserInfoResults } from '@api/loadUserInfo'
+interface Props {
+  router: NextRouter
+}
+
+export default withRouter(function StudentID({ router }: Props) {
+  const [studentID, setStudentID] = useState('')
+  const [disabled, setDisabled] = useState(true)
+  const { push } = useRouter()
```

Comment:
router from the props should have push method. Keep only one of them (`withRouter` or `useRouter`)